bot/exts/info/information.py

```diff
@@ -533,21 +553,43 @@ async def rules(self, ctx: Context, rules: Greedy[int]) -> None:
             await ctx.send(embed=rules_embed)
             return
 
-        full_rules = await self.bot.api_client.get("rules", params={"link_format": "md"})
-
         # Remove duplicates and sort the rule indices
-        rules = sorted(set(rules))
+        rule_numbers = sorted(set(rule_numbers))
+        # Remove duplicate keywords and preserve the order of initial keywords
+        keywords = list(dict.fromkeys(keywords))
```

Comment:
This isn't strictly necessary as you remove duplicates and sort later in `final_rule_numbers`